apps/condo/domains/user/schema/GetAccessTokenByUserIdService.js

```
@@ -58,18 +61,17 @@ const GetAccessTokenByUserIdService = new GQLCustomSchema('GetAccessTokenByUserI
-                    }
-                    throw new GQLError(ERRORS.ERROR_GETTING_ACCESS_TOKEN, context)
+                const { data: { userId, organizationId } } = args
+                if (!uuidValidate(userId)) {
+                    throw new GQLError(ERRORS.INVALID_USER_ID, context)
+                }
+                if (!uuidValidate(organizationId)) {
+                    throw new GQLError(ERRORS.INVALID_ORGANIZATION_ID, context)
+                }
+                const { error, accessToken, ttl } =  await getAccessTokenForUser(userId, organizationId)
+                if (error) {
+                    const errorKey = ERRORS.hasOwnProperty(error) ? error : 'ERROR_GETTING_ACCESS_TOKEN'
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)